ouroboros-consensus/src/Ouroboros/Consensus/Storage/ChainDB/API.hs

```diff
@@ -332,7 +333,20 @@ instance DB (ChainDB m blk) where
 -------------------------------------------------------------------------------}
 
 data AddBlockPromise m blk = AddBlockPromise
-    { blockProcessed          :: STM m (Point blk)
+    { blockWrittenToDisk      :: STM m Bool
```

Comment:
One other minor comment: you've organized the new promise around "written to disk". Based on the code I'm seeing, it looks like "added to VolDB" would also be accurate. Just pointing that out, in case the less-specific name is appealing.

Reply:
I take your point. But IMO "written to disk" is fine.